contracts/modules/LoanClosingsShared.sol

```diff
@@ -37,6 +38,25 @@ contract LoanClosingsShared is
 
     enum CloseTypes { Deposit, Swap, Liquidation }
 
+    /** modifier for invariant check */
+    modifier iTokenSupplyUnchanged(bytes32 loanId) {
+        Loan storage loanLocal = loans[loanId];
+        LoanParams storage loanParamsLocal = loanParams[loanLocal.loanParamsId];
+
+        address loanTokenPoolAddress = underlyingToLoanPool[loanParamsLocal.loanToken];
```

Comment:
is this necessary in 2 steps? can't we just read the lender from the loanLocal object?

Reply:
good point, just realized there is lender property in the struct.
Updated